h/static/scripts/controllers/character-limit-controller.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+function CharacterLimitController(element) {
+  var maxlength = parseInt(element.getAttribute('data-maxlength'));
+  var counter = element.parentElement.querySelector('.js-character-counter');
+
+  counter.innerHTML = '';
```

Comment:
If you just want to set the text of a node, use `Element#textContent`, which avoids the risk of introducing a vector for loading arbitrary HTML into the page.